### PR TITLE
[thread-cert] refactor case 9.2.18 using pktverify

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py [hunk 1]
```diff
@@ -170,6 +178,28 @@ def test(self):
...
+        COMMISSIONER = pv.vars['COMMISSIONER']
+        ROUTER_1 = pv.vars['ROUTER_1']
+        SED = pv.vars['SED']
+
+        # Step 1: Ensure the topology is formed correctly
+        pkts.filter_wpan_src64(ROUTER_1).filter_wpan_dst64(SED).filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
+
+        # Step 3: Leader MUST send MGMT_ACTIVE_SET.rsp (Accept) to the Commissioner
+        pkts.filter_wpan_src64(LEADER).filter_coap_ack(MGMT_ACTIVE_SET_URI).must_next().must_verify(
+            lambda p: bytes.fromhex('01') in p.coap.payload)
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py:195]: is it possible not to use `p.coap.payload`?

File: tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py [hunk 2]
```diff
@@ -170,6 +178,28 @@ def test(self):
...
+        LEADER = pv.vars['LEADER']
+        COMMISSIONER = pv.vars['COMMISSIONER']
+        ROUTER_1 = pv.vars['ROUTER_1']
+        SED = pv.vars['SED']
+
+        # Step 1: Ensure the topology is formed correctly
+        pkts.filter_wpan_src64(ROUTER_1).filter_wpan_dst64(SED).filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
+
+        # Step 3: Leader MUST send MGMT_ACTIVE_SET.rsp (Accept) to the Commissioner
+        pkts.filter_wpan_src64(LEADER).filter_coap_ack(MGMT_ACTIVE_SET_URI).must_next().must_verify(
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py:194]: Better to check the destination address?

File: tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py [hunk 3]
```diff
@@ -170,6 +178,28 @@ def test(self):
...
+            lambda p: bytes.fromhex('01') in p.coap.payload)
+
+        # Step 5: The Leader MUST send MGMT_PENDING_SET.rsp (Reject) to the Commissioner
+        pkts.filter_wpan_src64(LEADER).filter_coap_ack(MGMT_PENDING_SET_URI).must_next().must_verify(
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py:198]: Better to check the destination address?

File: tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py [hunk 4]
```diff
@@ -170,6 +178,28 @@ def test(self):
...
+            lambda p: bytes.fromhex('ff') in p.coap.payload)
+
+        # Todo: parse packets with non-default masterkey
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py:201]: Currently `pktverify` sets the masterkey and other configurations to `wireshark`. 
I think it's possible to configure multiple masterkeys to wireshark. 

https://github.com/openthread/openthread/blob/babb66d9c7cde0f3b8902a830fcb7de8294ccaac/tests/scripts/thread-cert/pktverify/consts.py#L173-L181

Can you check if it's doable?

File: tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py (hunk 1 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py]: use thread_meshcop.tlv.state instead

File: tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py (hunk 2 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py]: checked with COMMISSIONER_RLOC

File: tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py (hunk 3 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py]: checked with COMMISSIONER_RLOC

File: tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py (hunk 4 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py]: added multi key

File: tests/scripts/thread-cert/pktverify/consts.py [hunk 5]
```diff
@@ -213,12 +219,19 @@
...
+        'fd00:7d03:7d03:7d03::/64',
+    'wpan.802154_fcs_ok':
+        'FALSE',
+    'wpan.802154_sec_suite':
+        'AES-128 Encryption, 32-bit Integrity Protection',
+    'thread.thr_seq_ctr':
+        '00000000',
+    'uat:ieee802154_keys':
+        '''"00112233445566778899aabbccddeeff","1","Thread hash"
+                              "ffeeddccbbaa99887766554433221100","2","Thread hash"''',
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/pktverify/consts.py:234]:
```suggestion
                              "ffeeddccbbaa99887766554433221100","1","Thread hash"''',
```

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1: 

with a nit

**Review by jwhui — APPROVED** [review]

**Pull request merged**